css/style.css

```diff
@@ -758,6 +1157,19 @@
     color: #fff;
     font-size: 16px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 767px;
+  }
+
+  .info__title {
+    font-size: 62px;
+    text-align: right;
+    padding: 0 100px 30px 0;
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)